package.json

```diff
@@ -11,10 +11,11 @@
   },
   "main": "index.js",
   "scripts": {
-    "prepare": "npm prune && npm run build",
-    "build": "rm -rf lib && babel src --out-dir lib -s > /dev/null",
+    "prepare": "rm -rf ./lib",
+    "postinstall": "npm run build",
+    "build": "babel ./src --out-dir ./lib -s",
```

Comment:
I had to remove `> /dev/null` part, so it will be verbose in the command line (but I think that's not an issue), because on Windows the `npm install` was failing. This was because of https://github.com/babel/babel/issues/9871#issuecomment-484409034

Reply:
I see, didn't realize that device did not exist on windows :)